src/app.jsx

```diff
@@ -20,7 +21,22 @@ const PageTypes = {
  */
 const App = ({ resource, problem }) => {
   if (problem) {
-    return (<div><h1>Error</h1><p>An unexpected problem occurred.</p></div>);
+    let text = 'Something went terribly, terribly wrong.';
```

Comment:
What do you think about moving this text into the component and using a shorter construction like:

```
const phrases = {
  403: 'Sign in or get out!',
  404: "Oops, this is not the page you're looking for.",
};

const text = phrases?.[status] || 'Something went terribly, terribly wrong.';